src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
need to add class like that and make possibility to control of number of days and start day by changing this class 
```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-day--wed">
```